plugins/woocommerce-blocks/assets/js/blocks/product-gallery/save.tsx

```diff
@@ -4,24 +4,9 @@
 import { useBlockProps, useInnerBlocksProps } from '@wordpress/block-editor';
 import clsx from 'clsx';
 
-/**
- * Internal dependencies
- */
-import { getClassNameByNextPreviousButtonsPosition } from './utils';
-import { ProductGalleryAttributes } from './types';
-
-export const Save = ( {
-	attributes,
-}: {
-	attributes: ProductGalleryAttributes;
-} ): JSX.Element => {
+export const Save = (): JSX.Element => {
```

Comment:
This type is redundant

```suggestion
export const Save = () => {
```